packages/slate/src/interfaces/editor.ts

```diff
@@ -603,7 +603,7 @@ export const Editor: EditorInterface = {
       return cachedIsEditor
     }
 
-    if (!isPlainObject(value)) {
+    if (typeof value !== 'object') {
```

Comment:
I kept this simpler check here, because we try to use the value as a WeakMap key on line 631, which will crash if a primitive value slips through.

Reply:
It's probably safe, though this one typeof compared to the myriad down below doesn't seem like it'll gain much for us